runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -160,8 +160,8 @@ getReturnTypeFromMetaData(J9UpcallMetaData *data)
 {
 	J9JavaVM *vm = data->vm;
 	J9VMThread *currentThread = currentVMThread(vm);
-	j9object_t methodType = J9VMOPENJ9INTERNALFOREIGNABIUPCALLMHMETADATA_CALLEETYPE(currentThread,
-			J9_JNI_UNWRAP_REFERENCE(data->mhMetaData));
+	j9object_t methodType = J9VMJAVALANGINVOKEMETHODHANDLE_TYPE(currentThread,
+			J9VMOPENJ9INTERNALFOREIGNABIUPCALLMHMETADATA_CALLEEMH(currentThread, J9_JNI_UNWRAP_REFERENCE(data->mhMetaData)));
 	J9Class *retTypeClass = J9VM_J9CLASS_FROM_HEAPCLASS(currentThread,
 			J9VMJAVALANGINVOKEMETHODTYPE_RTYPE(currentThread, methodType));
```

Comment:
Each argument should begin on a new line.

Reply:
Fixed.